config/config.go

```diff
@@ -288,11 +288,11 @@ type TiKVClient struct {
 
 // Binlog is the config for binlog.
 type Binlog struct {
-	Enable       bool   `toml:"enable" json:"enable"`
-	WriteTimeout string `toml:"write-timeout" json:"write-timeout"`
+	Enable bool `toml:"enable" json:"enable"`
 	// If IgnoreError is true, when writing binlog meets error, TiDB would
 	// ignore the error.
-	IgnoreError bool `toml:"ignore-error" json:"ignore-error"`
+	IgnoreError  bool   `toml:"ignore-error" json:"ignore-error"`
+	WriteTimeout string `toml:"write-timeout" json:"write-timeout"`
```

Comment:
IMHO, sizeof(`string`) is bigger than bool?

Reply:
The adjustment criterion is: **8 bytes should be arranged as full as possible**.
For this structure, the first two bool account for 2 byte and 6 byte padding.
No matter this 8byte is placed at the front or back of the structure, the memory usage is the same.

Reply:
The adjustment criterion is: **8 bytes should be arranged as full as possible**.
For this structure, the first two bool account for 2 byte and 6 byte padding.
No matter this 8byte is placed at the front or back of the structure, the memory usage is the same.

Reply:
string consist of char* and length